uiowa/uiowa
# Card-block-form.js machine name updates

Related to https://github.com/uiowa/uiowa/issues/7221 but does not resolve that issue. 

In https://github.com/uiowa/uiowa/commit/e5c62aadc2df3df19c717df3b20bc9571c73a9d7, we updated the media size style and missed changing the machine name in a few spots. 

# How to test

```
ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /node/1221/layout
```
- Inspect source on the layout builder styles and verify that the machine name change updates are correct
- Set media position to stacked and circle small, then attach a video and save the card
- Confirm in the html source that `media--large` class is added
- Run the same steps on prod and see that the `media--small` class is added
